src/status_im/contexts/wallet/bridge/confirm_transaction/view.cljs

```diff
@@ -11,7 +11,7 @@
   []
   [rn/view {:style style/bridge-send-wrapper}
    [input-amount/view
```

Comment:
I am confused why this is not `bridge.input-amount`?

Reply:
I must have got it backwards 🙃 

I'll adjust!